README.md

```diff
@@ -1,30 +1,27 @@
 # API Linter
+
 API linter checks APIs defined in protobuf files. It follows [Google API Design Guide](https://cloud.google.com/apis/design/).
 
 ## Requirements
+
 * Install `git` from [https://git-scm.com](https://git-scm.com/);
 * Install `go` from [https://golang.org/doc/install](https://golang.org/doc/install);
 * Install `protoc` by following this [guide](http://google.github.io/proto-lens/installing-protoc.html);
 
 ## Installation
-* Download `api-linter` using `git`:
-```sh
-git clone git@github.com:googleapis/api-linter.git $HOME/Downloads/api-linter
-```
-or
-```
-git clone https://github.com/googleapis/api-linter.git $HOME/Downloads/api-linter
-```
-* Install `api-linter` using `go install`:
+
+* Install `api-linter` using `go get`:
+
 ```sh
-cd $HOME/Downloads/api-linter/cmd/api-linter
-go install
+go get github.com/googleapis/api-linter/cmd/api-linter
 ```
-* Ensure that your operating system's `PATH` contains `go install`ed binaries, as described in the
-  [Go install guide](https://golang.org/doc/install#install).
```

Comment:
Should we keep this link? Also, I don't think `$HOME/go/bin` is always the right location. For example in [the Linux / Mac instructions](https://golang.org/doc/install#tarball), it recommends `/usr/local/go`

Reply:
I believe /usr/local/go/bin contains the go binaries, but not the ones installed by a user through `go get`, which will be in `$GOPATH/bin`. By default, `$GOPATH` is `$HOME/go`. If a user sets `$GOPATH`, he/she is experienced enough to figure out where the go binaries are installed.

See https://golang.org/doc/install#testing

Reply:
Ah I see, makes sense.